app/controllers/api/base_controller.rb

```diff
@@ -1,26 +1,78 @@
 class Api::BaseController < ApplicationController
   class SchemaValidationError < StandardError ; end
 
-  def render_schema_json(data, schema: nil, **kwargs)
```

Comment:
I'm removing the `schema` keyword argument. I think it's better for each action to have its own dedicated schema.